crates/router/src/connector/multisafepay/transformers.rs

```diff
@@ -0,0 +1,453 @@
+    pub browser: Option<Browser>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub locale: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub ip_address: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub forward_ip: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub first_name: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub last_name: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub gender: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub birthday: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub address1: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub address2: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub house_number: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub zip_code: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub city: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub state: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub country: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub phone: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub email: Option<masking::Secret<String, Email>>, 
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub user_agent: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub referrer: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub reference: Option<String>
+}
+
+#[derive(Clone, Debug, Eq, PartialEq, Deserialize, Serialize)]
+pub struct GatewayInfo {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub card_number: Option<Secret<String, pii::CardNumber>>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub card_holder_name: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub card_expiry_date: Option<i32>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub card_cvc: Option<Secret<String>>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub flexible_3d: Option<bool>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub moto: Option<bool>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub term_url: Option<String>
+}
+
+#[derive(Clone, Debug, Eq, PartialEq, Deserialize, Serialize)]
+pub struct CheckoutOptions {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub validate_cart: Option<bool>
+}
+
+//TODO: Fill the struct with respective fields
+#[derive(Clone, Debug, Eq, PartialEq, Deserialize, Serialize)]
+pub struct MultisafepayPaymentsRequest {
+    #[serde(rename = "type")]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub _type: Option<String>,
```

Comment:
Do not use fields names starting with underscore, provide meaningful name for the field